src/Microsoft.AspNetCore.Razor.Language/Legacy/IParentChunkGenerator.cs

```diff
@@ -5,9 +5,5 @@ namespace Microsoft.AspNetCore.Razor.Language.Legacy
 {
     internal interface IParentChunkGenerator
```

Comment:
>IParentChunkGenerator [](start = 23, length = 21)

Can we remove this? Or still needed as a marker type?

Reply:
I think we only need ISpanChunkGenerator as the marker type. I'll check and make sure.

---
In reply to: [233615835](https://github.com/aspnet/Razor/pull/2590#discussion_r233615835) [](ancestors = 233615835)